.eslintrc.js

```diff
@@ -1,20 +1,13 @@
 module.exports = {
-  root: true,
-  parser: '@typescript-eslint/parser',
   extends: [
     'eslint:recommended',
+    'next/core-web-vitals',
```

Comment:
This `next/core-web-vitals` preset seems to replace the old `next` preset, and comes with some stricter rules, I think?

Quote from Next.js docs that cover ESLint setup [^docs]:

> **Strict**: Includes Next.js' base ESLint configuration along with a stricter Core Web Vitals rule-set. This is the recommended configuration for developers setting up ESLint for the first time.
> 
> ```json
> {
>   "extends": "next/core-web-vitals"
> }
> ```
> 

[^docs]: https://nextjs.org/docs/basic-features/eslint#eslint-plugin

Reply:
Yes, basically this preset enables the basic `next` one plus some other rules for core web vitals. This is a more extended preset.